File: overridding.java
class PClass {
    public int i = 100;

    public void show() {
        System.out.println("hello P");
    }
}

class CClass extends PClass {
    public int i = 200; 

    @Override
    public void show() {
        System.out.println("hello C");
    }
}
class HelloWorld {
    public static void main(String[] args) {
        PClass pobj = new CClass();
        System.out.println(pobj.i);
    }
}

File: overridding.cs
using System;

class PClass {
    public int i = 100;
    public virtual void show(){
        Console.WriteLine("hello P");
    }
}

class CClass : PClass {
    public new int i = 200; 
    public override void show(){
        Console.WriteLine("hello C");
    }
}

public class HelloWorld
{
    public static void Main(string[] args)
    {
        PClass pobj = new CClass();
        pobj.show();
        Console.WriteLine(pobj.i); 
    }
}
